packages/core/src/temporal/IsoFields.js

```diff
@@ -812,7 +812,7 @@ class Unit extends TemporalUnit {
     }
 
     toString() {
-        return name;
+        return this._name;
```

Comment:
nice bugfix :+1: What about a test for it ?

Reply:
Can do! ;)